src/Compilers/CSharp/Portable/Syntax/SyntaxFacts.cs

```diff
@@ -539,8 +539,14 @@ internal static bool HasYieldOperations(SyntaxNode? node)
         {
             // Do not descend into functions and expressions
             return node is object &&
-                   node.DescendantNodesAndSelf(child => !IsNestedFunction(child) && !(node is ExpressionSyntax)).
-                       OfType<YieldStatementSyntax>().Any();
+                   node.DescendantNodesAndSelf(child => !IsNestedFunction(child) && !(node is ExpressionSyntax)).Any(n => n is YieldStatementSyntax);
+        }
+
+        internal static bool HasReturnWithExpression(SyntaxNode? node)
```

Comment:
>SyntaxNode? [](start = 53, length = 11)

When is this `null`?

Reply:
> When is this null?

When the method is called with one.

---
In reply to: [410424982](https://github.com/dotnet/roslyn/pull/43451#discussion_r410424982) [](ancestors = 410424982)